digits/templates/home.html

```diff
@@ -89,7 +89,7 @@ <h4 class="list-group-item-heading"><a href="{{ url_for('datasets_show', job_id=
         <ul class="list-group">
             {% for job in batch %}
             <li class="list-group-item">
-            <a class="btn btn-xs btn-danger pull-right" onClick="return deleteJob('{{job.id()}}');">Delete</a>
+                <a class="btn btn-xs btn-danger pull-right" onClick="return deleteJob('{{job.id()}}');">Delete</a>
```

Comment:
While you're at it, can you move any whitespace changes that you feel are necessary into a separate commit, too? It should be relatively easy.
```
git rebase origin/master
git reset origin/master
git add -p
```